src/Raven.Server/Utils/AsyncBlittableJsonTextWriterForDebug.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.IO;
+using Microsoft.AspNetCore.Http;
+using Raven.Server.Extensions;
+using Raven.Server.ServerWide;
+using Sparrow.Json;
+
+namespace Raven.Server.Utils
+{
+    public class AsyncBlittableJsonTextWriterForDebug : AsyncBlittableJsonTextWriter
+    {
+        private readonly ServerStore _serverStore;
+        private bool _isFirst = true;
+        private bool _isOnlyWrite;
+        private bool _skipDebug;
+
+        public AsyncBlittableJsonTextWriterForDebug(HttpRequest request, JsonOperationContext context, ServerStore serverStore, Stream stream) : base(context, stream)
+        {
+            _serverStore = serverStore;
+            _skipDebug = request.IsFromClientApi() || request.IsFromStudio();
```

Comment:
Since we reverted the `Identities` endpoint, this won't be needed anymore.

Reply:
done